terraform/main.tf

```diff
@@ -0,0 +1,9 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+# https://www.terraform.io/docs/providers/aws/
+provider "aws" {
+  region = "us-west-2"
```

Comment:
We generally like to set  this as 

```
provider "aws" {
   region = var.region
}
```

And then have a `variables.tf` file and set a default value to the region variable to `us-west-2` this is still ok though